docs/changelogs/v0.26.md

```diff
@@ -0,0 +1,29 @@
+# Kubo changelog v0.26
+
+- [v0.26.0](#v0260)
+
+## v0.26.0
+
+- [Overview](#overview)
+- [🔦 Highlights](#-highlights)
+  - [Several deprecated commands have been removed](#several-deprecated-commands-have-been-removed)
+- [📝 Changelog](#-changelog)
+- [👨‍👩‍👧‍👦 Contributors](#-contributors)
+
+### Overview
+
+### 🔦 Highlights
+
+#### Several deprecated commands have been removed
+
+Several deprecated commands have been removed:
+
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
```

Comment:
:warning:  @hacdias what is the alternative to `ipfs urlstore add` ?
I know for filestore we can use `ipfs add --nocopy`, but does the same apply here? If so, mind updating  https://github.com/ipfs/kubo/blob/master/docs/experimental-features.md#ipfs-urlstore too?

If not, we need to keep this.

```suggestion
- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535), use {TODO} instead
```

Reply:
Yes, it does exactly the same. It's documented. I will update.